### PR TITLE
OHRM-175: Add market place plugin

### PR DESCRIPTION
**Pull request by AshanKooragoda**

### CODE REVIEW

File: symfony/plugins/orangehrmMarketPlacePlugin/config/orangehrmMarketPlacePluginConfiguration.class.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * Boston, MA 02110-1301, USA
+ */
+
+/**
+ * Class orangehrmMarketPlacePluginConfiguration
+ */
+class orangehrmMarketPlacePluginConfiguration extends sfPluginConfiguration
+{
+    /**
+     * @return bool|null|void
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmMarketPlacePlugin/config/orangehrmMarketPlacePluginConfiguration.class.php]: is this method return bool?

File: symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     * @return array
+     */
+    protected function getAddonsFromMP()
+    {
+        $addons = array();
+        return $addons;
+    }
+
+    /**
+     * @param $addonId
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php]: @param int $addonId

File: symfony/plugins/orangehrmMarketPlacePlugin/test/model/service/APIManagerServiceTest.php [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA
+ */
+
+/**
+ * Class APIManagerServiceTest
+ */
+class APIManagerServiceTest extends PHPUnit_Framework_TestCase
+{
+    /**
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmMarketPlacePlugin/test/model/service/APIManagerServiceTest.php]: no need of empty doc block

File: symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php (hunk 2 above)
Comment (AshanKooragoda) [reply symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php]: doc comment will be corrected.

File: symfony/plugins/orangehrmMarketPlacePlugin/test/model/service/APIManagerServiceTest.php (hunk 3 above)
Comment (AshanKooragoda) [reply symfony/plugins/orangehrmMarketPlacePlugin/test/model/service/APIManagerServiceTest.php]: removed

File: symfony/plugins/orangehrmMarketPlacePlugin/config/orangehrmMarketPlacePluginConfiguration.class.php (hunk 1 above)
Comment (samanthajayasinghe) [reply symfony/plugins/orangehrmMarketPlacePlugin/config/orangehrmMarketPlacePluginConfiguration.class.php]: no matter .. it's wrong and you committed it..

Comment (AshanKooragoda) [reply symfony/plugins/orangehrmMarketPlacePlugin/config/orangehrmMarketPlacePluginConfiguration.class.php]: change it

**Pull request merged**